README.md

```diff
@@ -1,13 +1,60 @@
-# playground
+# Vlang Playground
 
-Source code for V's playground.
+This repository container the source code for V's playground.
```

Comment:
This repository contains the source code for V's playground.

Reply:
Haha sorry, was the autocomplete, done!